codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpBoundProtocolGenerator.kt

```diff
@@ -87,6 +89,12 @@ import java.util.logging.Logger
 sealed class ServerHttpBoundProtocolSection(name: String) : Section(name) {
     data class AfterTimestampDeserializedMember(val shape: MemberShape) :
         ServerHttpBoundProtocolSection("AfterTimestampDeserializedMember")
+
+    data class TypeOfSerializedPayloadStream(val params: StreamPayloadSerializerParams) :
+        ServerHttpBoundProtocolSection("TypeOfSerializedPayloadStream")
+
+    data class WrapStreamAfterPayloadGenerated(val params: StreamPayloadSerializerParams) :
+        ServerHttpBoundProtocolSection("WrapStreamAfterPayloadGenerated")
```

Comment:
Add comments explaining these and how to implement them.

Reply:
Addressed in 3db0525.